### DISCUSSION TITLE
Inconsistent Coordinates between Compute2DCoords and Draw

### DISCUSSION THREAD

**Discussion by bwolfe-benchling** [rdkit/rdkit#6323] [General]
When trying to assess the quality of coordinates generated for molecule images I found that `COC1=CC=C(C[C@@H]2NC(=O)[C@@H](CC3=CNC4=CC=CC=C34)NC(=O)[C@H](CC3=CC=CC=C3)NC(=O)[C@H]3CCCN3C2=O)C=C1` seemingly generates different coordinates when using `rdDepictor.Compute2DCoords ` or `AllChem.Compute2DCoords` than it does when using `Draw.MolsToImage` or `d2d.DrawMolecule`

![image](https://user-images.githubusercontent.com/82657751/234930950-6b14a0c0-dea6-4eb5-aea7-f4152da70ed2.png)

How can I assess the quality of the image I am generating if the coordinates I generate with `Compute2DCoords` are not what I will be seeing in the generated image?  I looked at https://github.com/rdkit/rdkit/discussions/4150 but it is unclear if this mismatch in coordinate generation is expected.

**Comment by greglandrum** [comment]
@bwolfe-benchling this is definitely a bug in the RDKit. You should get the same result for all of these.
Pleaes create an issue from this (there should be a link to the right  to do it) and we'll look into it.
